src/PurgeClient/VarnishPurgeClient.php

```diff
@@ -53,7 +53,7 @@ public function purge($tags)
 
             $headers = [
                 'key' => $tag,
-                'Host' => empty($_SERVER['SERVER_NAME']) ? 'localhost' : $_SERVER['SERVER_NAME'],
```

Comment:
`localhost` is not a good fallback. Won't work on Platform.sh ( and other setups to ) from CLI where purge server is not local host